src/QuickMarcEditor/QuickMarcEditorRows/SearchLink/SearchLink.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import flatten from 'lodash/flatten';
+
+import { IconButton } from '@folio/stripes/components';
+import { advancedSearchQueryBuilder } from '@folio/stripes-inventory-components';
+
+import { getContentSubfieldValue } from '../../utils';
+
+const propTypes = {
+  field: PropTypes.shape({
+    content: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+const SearchLink = ({ field }) => {
+  const content = getContentSubfieldValue(field.content);
+
+  const numberOfAdvancedSearchRows = 6;
+  // we search by field's $a and $z values
+  const valuesToSearchBy = flatten([content.$a, content.$z]).slice(0, numberOfAdvancedSearchRows);
+
+  const advancedSearchRows = valuesToSearchBy.filter(value => Boolean(value)).map(value => ({
+    bool: 'or',
+    query: value,
+    match: 'exactPhrase',

```

Comment:
Let's use constants from `stripes-components`. It will be easier if someone decides to change `exactPhrase` to something else.